### ISSUE TITLE
installing tsdl-image and tsdl-mixer on a mac

### ISSUE THREAD

**Issue by sanette** [ocaml/opam-repository#20268]
it seems that users have difficulties to install `conf-sdl2-image` or `conf-sdl2-mixer` on a Mac.
The situation seems much better with `conf-sdl2-ttf`.
Having a look at https://github.com/ocaml/opam-repository/blob/master/packages/conf-sdl2-ttf/conf-sdl2-ttf.1/opam
and https://github.com/ocaml/opam-repository/blob/master/packages/conf-sdl2-image/conf-sdl2-image.1/opam
we see that the latter has much less options than the former.

I don't have a mac to test it myself, could someone confirm that one can copy the `ttf` config to the `image` config?
(I assume it is enough to replace everywhere `ttf` by `image`?)
(same for `mixer`, of course)

**Comment by sanette** [comment]
pinging @mbacarella @toots

**Comment by toots** [comment]
As far as I can tell `conf-sdl2-image` is missing a `depext` for `macos`. Using brew, the package should be `sdl2_image`. I'd recommend investigating the depext for the other OS/platforms listed in `conf-sdl2-ttf` while you're at it.

**Comment by sanette** [comment]
partially resolved by https://github.com/ocaml/opam-repository/pull/20270

**Comment by mbacarella** [comment]
Not quite sure if this is what you mean, but if I do this

```
opam source conf-sdl2-image
cd conf-sdl2-image.1
# add  to opam depexts:
#  ["sdl2_image"] {os-distribution = "homebrew" & os = "macos"} to the opam file
opam install .
```

it works.

Same if I do something similar with `conf-sdl2-mixer`.

(Though I have to run `opam install .` twice for each case due to caching or whatever)

**Comment by sanette** [comment]
yes! so now, after doing `opam update; opam upgrade`, I assume that if you try `opam install conf-sdl2-image`, or simply `opam install tsdl-image`, 
it should work

**Comment by mbacarella** [comment]
Yup, works.
